### ISSUE TITLE
Allow listing more than 10 snapshots for an environment

### ISSUE THREAD

**Issue by budda** [platformsh/platformsh-cli#348]
Feature request.

→ platform snapshot:list --project u3tucrjygkrgk --environment snapshot

The above only gives me 10 snapshots with their dates. We may wish to restore a snapshot from 3 months ago so need a way to get snapshots list for a given date range or just a full list maybe?

The Web UI doesn't help find snapshots over 30 days old either as everything is shown as "a month ago" until you manually hover over it with the mouse (very time consuming!)

**Comment by budda** [comment]
p.s. worth noting that ```platform snapshot:list --project u3tucrjygkrgk --environment master --limit=100``` doesn't return more than 10, but a limit of 5 does return a smaller list.

**Comment by pjcdawkins** [comment]
Yes, this is a higher up bug (in the API) which we are tracking [internally](https://platformsh.atlassian.net/browse/PF-83). At the moment there is a limit effectively hardcoded to 10. I'm leaving this issue open as it provides more visibility on the bug.

**Comment by budda** [comment]
How does the web UI manage to keep loading more snapshots as you scroll down the event list if it's using the same api?

**Comment by pjcdawkins** [comment]
Good point - there is a `starts_at` parameter on the API, which takes a timestamp.

```
GET /api/projects/<projectId>/environments/<environmentId>/activities?starts_at=2015-09-25T11:47:46.506908%2B00:00
```

Let me investigate how the web UI uses that. It doesn't seem very practical to specify times on the command-line but perhaps it could page through in a similar infinite-scroll kind of way, at least.

**Comment by pjcdawkins** [comment]
OK... I've added a `--start` option to `snapshot:list`. Specify any date/time that PHP's `strtotime()` understands.

The `--limit` option still won't work. I'll change this issue's title to reflect this.
